lib/ansible/modules/cloud/docker/docker_swarm_info.py

```diff
@@ -305,6 +325,10 @@ def get_essential_facts_services(item):
 
         return object_essentials
 
+    def get_docker_swarm_unlock_key(self):
+        unlock_key = self.client.get_unlock_key()
```

Comment:
```suggestion
        unlock_key = self.client.get_unlock_key() or {}
```
Just to avoid that the next line fails.